### ISSUE TITLE
Update nanopb dependency version for PrivacyManifest support

### ISSUE THREAD

**Issue by tosh7** [google-ar/arcore-ios-sdk#97]
This request pertains to updating the version constraint of the nanopb dependency utilized within the ARCore project. ARCore includes a dependency on [nanopb](https://github.com/nanopb/nanopb), which is listed in [Apple's PrivacyManifest third-party SDK list](https://developer.apple.com/support/third-party-SDK-requirements/).

To incorporate the version of nanopb that supports PrivacyInfo.xcprivacy, it is necessary to use versions later than [2.30910.0](https://github.com/nanopb/nanopb/releases/tag/0.3.9.10).

Could you please update the ARCore.podspec.json and Package.swift files accordingly? The suggested changes are as follows:

For ARCore.podspec.json, update the nanopb version range to:

```
"nanopb": [
  ">= 2.30908.0",
  "< 2.30911.0"  // Adjusted to include versions later than 2.30910.0
]
```

For Package.swift, modify the nanopb package version range to:

```
.package(url: "https://github.com/firebase/nanopb.git", "2.30909.0"..<"2.30911.0") // Adjusted to include versions later than 2.30910.0
```
These adjustments would enable the use of nanopb versions that support PrivacyInfo.xcprivacy, aligning with the requirements listed in Apple's third-party SDK guidelines.

Thank you for considering this request. Your assistance in maintaining compatibility with the latest standards is greatly appreciated.

**Comment by 15kingben** [comment]
This will be updated in the next ARCore release

**Comment by jrullman** [comment]
This is fixed in ARCore 1.44.
